ui/components/AppMasthead.tsx

```diff
@@ -80,24 +85,39 @@ export function AppMasthead({
           >
             <ToolbarContent id={"masthead-toolbar"}>
               <ToolbarGroup
-                variant="icon-button-group"
-                align={{ default: "alignRight" }}
-                spacer={{ default: "spacerNone", md: "spacerMd" }}
+                variant="action-group"
+                align={{ default: "alignEnd" }}
               >
+                <ToggleGroup className={"pf-v6-u-py-sm"}>
+                  <ToggleGroupItem
+                    icon={<SunIcon />}
+                    aria-label="Light mode"
+                    isSelected={!isDarkMode}
+                    onChange={() => {
+                      toggleDarkMode(false);
+                    }}
+                  />
+                  <ToggleGroupItem
+                    icon={<MoonIcon />}
+                    aria-label="Dark mode"
+                    isSelected={isDarkMode}
+                    onChange={() => {
+                      toggleDarkMode(true);
+                    }}
+                  />
+                </ToggleGroup>
                 <ToolbarGroup
-                  variant="icon-button-group"
+                  variant="label-group"
                   visibility={{ default: "hidden", lg: "visible" }}
                 >
-                  {isTechPreview && (
-                    <ToolbarItem>
-                      <TechPreviewPopover>
-                        <Label color={"blue"} isCompact={true}>
-                          {t("AppMasthead.tech_preview_label")}
-                        </Label>
-                      </TechPreviewPopover>
-                    </ToolbarItem>
-                  )}
-                  <ToolbarItem>
+                  <ToolbarItem className={"pf-v6-u-py-sm"}>
+                    <TechPreviewPopover>
+                      <Label color={"blue"} isCompact>
+                        {t("AppMasthead.tech_preview_label")}
+                      </Label>
+                    </TechPreviewPopover>
+                  </ToolbarItem>
```

Comment:
It looks like we lost the ability to turn this off. IMO we can remove this out entirely now anyway.